awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvListItem.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { string } from 'prop-types';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Link } from 'react-router-dom';
+import {
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells,
+} from '@patternfly/react-core';
+
+import DataListCell from '../../../components/DataListCell';
+import { ExecutionEnvironment } from '../../../types';
+
+function OrganizationExecEnvListItem({
+  executionEnvironment,
+  detailUrl,
+  i18n,
+}) {
+  const labelId = `check-action-${executionEnvironment.id}`;
+
+  return (
+    <DataListItem
+      key={executionEnvironment.id}
+      aria-labelledby={labelId}
+      id={`${executionEnvironment.id} `}
+    >
+      <DataListItemRow>
+        <DataListItemCells
+          dataListCells={[
+            <DataListCell
+              key="image"
+              aria-label={i18n._(t`execution environment image`)}
```

Comment:
Possibly need some sentence casing here? 
```suggestion
              aria-label={i18n._(t`Execution environment image`)}
```